src/app/services/graph-client/msal-service.ts

```diff
@@ -18,6 +18,7 @@ export function getSessionId() {
 export async function logIn(sessionId = ''): Promise<any> {
   const loginRequest: AuthenticationParameters = {
     scopes: defaultUserScopes,
+    prompt: 'select_account'
```

Comment:
if you only have one account does this cause you to have to select it each time?

Reply:
It is a safe bet... There could be a way to check whether there are other accounts stored in the cache... But I did not investigate this